src/binaryutils.cpp

```diff
@@ -186,13 +186,17 @@ Status validateResolutionAgainstFirstBatchImage(const cv::Mat input, cv::Mat* fi
 
 bool checkBatchSizeMismatch(const std::shared_ptr<TensorInfo>& tensorInfo,
     const int batchSize) {
-    if (batchSize <= 0)
+    auto d = tensorInfo->getBatchSize();
```

Comment:
Use better naming than `d`
`TensorInfo::getBatchSize` should return `Dimension` type